dsgrid/config/annual_time_dimension_config.py

```diff
@@ -0,0 +1,30 @@
+from datetime import timedelta
```

Comment:
Making one module per time config seems like it will swamp everything else in this sub-package. Should we

- Put all of the time configs in one time_dimension_configs.py file OR
- Make a configs/time subpackage

?

Reply:
@elainethale I think this item is resolved per our conversation, but let me know if you disagree.